site/app/models/User.php

```diff
@@ -418,6 +428,10 @@ public function setRegistrationSection($section) {
         $this->registration_section = ($section !== null) ? $section : null;
     }
 
+    public function setRegistrationSubsection($section) {
+        $this->registration_subsection = $section;
+    }
+
```

Comment:
you can get rid of this function, it's not needed (as using `@prop` annotation automatically makes available this setter field.